zookeeper-server/src/main/java/org/apache/zookeeper/server/DataTree.java

```diff
@@ -1962,4 +1962,18 @@ public static StatPersisted createStat(long zxid, long time, long ephemeralOwner
         stat.setEphemeralOwner(ephemeralOwner);
         return stat;
     }
+
+    // for test only
+    public static StatPersisted createStat(int version) {
```

Comment:
```suggestion
   static StatPersisted createStat(int version) {
```

Let's narrow the scope as possible.